packages/contract-method-name/eth-name.js

```diff
@@ -0,0 +1,59 @@
+import {MethodRegistry} from 'eth-method-registry'
+import {fetchWithStorage} from '@cfxjs/utils'
+import {
+  ETH_MAINNET_RPC_ENDPOINT,
+  ETH_ROPSTEN_RPC_ENDPOINT,
+  ETH_FOUR_BYTE_DOMAIN,
+} from '@cfxjs/fluent-wallet-consts'
+
+import Eth from 'ethjs'
+
+const getEthMethodName = async fourBytePrefix => {
+  const res = await fetchWithStorage(
+    'get',
+    `${ETH_FOUR_BYTE_DOMAIN}/api/v1/signatures/?hex_signature=${fourBytePrefix}`,
+    {
+      referrerPolicy: 'no-referrer-when-downgrade',
+      body: null,
+      mode: 'cors',
+    },
+  )
+
+  if (res?.count === 1) {
```

Comment:
I&rsquo;m not familiar with the 4byte api, is this line checking for exact match?

-   Is it possible that the api still returns one result without exact match?
-   Does the api supports checking exact match with url params?

Reply:
Yes. Here is this doc https://www.4byte.directory/docs/。It support exact match, but also return array